test/gerkin/tests/features/data_delete.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Remove record from the database.
+  Request endpoint: DELETE /data/{code}/{version}/delete
+
+  Background:
+    Given database with the valid schema has been set up
+
+  Scenario: Successfully delete existing record
+    When I make a DELETE request with a valid payload ID <ID>
+    Then I receive a HTTP 204 response
+
+  Scenario: Failure when delete existing record
```

Comment:
As there's no payload we can change this scenario to one in which record with given ID is not present om database